pkg/monitoring/metrics/template-validator/metrics.go

```diff
@@ -4,10 +4,17 @@ import (
 	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
 )
 
-func SetupMetrics() {
-	if err := operatormetrics.RegisterMetrics(
+func SetupMetrics() error {
+	err := operatormetrics.RegisterMetrics(
```

Comment:
same as for pkg/monitoring/metrics/ssp-operator/metrics.go

Reply:
fixed